core/templates/dev/head/css/oppia.css

```diff
@@ -53,8 +53,9 @@ md-input-group.md-default-theme label
   margin-bottom: 0;
 }
 
-.md-button-success {
```

Comment:
Don't overwrite this; we use it elsewhere. The command I used to verify that was:
```
    grep "md-button-success" . -r --exclude-dir=third_party
```